Test instability in `client_test.go`

This isn't a new issue, but for a while we're been seeing intermittent test failures. It happens on around once in every four runs, both locally and in CI:

```
--- FAIL: TestClient_do (18.02s)
    client_test.go:124: 
        	Error Trace:	client_test.go:124
        	Error:      	Expected nil, but got: &errors.errorString{s:"request failed after 5 retries with err: Post http://127.0.0.1:45783/assets.json: net/http: request canceled (Client.Timeout exceeded while awaiting headers)"}
        	Test:       	TestClient_do
    client_test.go:139: 
        	Error Trace:	client_test.go:139
        	Error:      	Expected nil, but got: &errors.errorString{s:"request failed after 5 retries with err: Post http://127.0.0.1:45783/assets.json: net/http: request canceled (Client.Timeout exceeded while awaiting headers)"}
        	Test:       	TestClient_do
FAIL
```

It's not critical but it would be nice to resolve this.

--- a/src/httpify/client_test.go
+++ b/src/httpify/client_test.go
@@ -113,7 +113,7 @@ func TestClient_do(t *testing.T) {
 
 	client, _ = NewClient(Params{
 		Domain:  server.URL,
-		Timeout: time.Millisecond,
+		Timeout: 5 * time.Millisecond,
 	})
 	client.baseURL.Scheme = "http"
 
@@ -130,7 +130,7 @@ func TestClient_do(t *testing.T) {
 
 	client, _ = NewClient(Params{
 		Domain:  server.URL,
-		Timeout: time.Millisecond,
+		Timeout: 5 * time.Millisecond,
 	})
 	client.maxRetry = 1
 	client.baseURL.Scheme = "http"

